test/acceptance/README.md

```diff
@@ -0,0 +1,116 @@
+# STH Acceptance Tests
+
+Folder for acceptance tests of STH (Short Term Historic).
+
+## How to Run the Acceptance Tests
+
+### Prerequisites:
+
+- Python 2.6 or newer
+- pip installed (http://docs.python-guide.org/en/latest/starting/install/linux/)
+- virtualenv installed (pip install virtualenv) (optional).
+Note: We recommend the use of virtualenv, because is an isolated working copy of Python which allows you to work on a specific project without worry of affecting other projects.
+
+##### Environment preparation:
+
+- If you are going to use a virtual environment (optional):
+  * Create a virtual environment somewhere, e.g. in ~/venv (virtualenv ~/venv) (optional)
+  * Activate the virtual environment (source ~/venv/bin/activate) (optional)
+- Both if you are using a virtual environment or not:
+  * Change to the test/acceptance folder of the project.
+  * Install the requirements for the acceptance tests in the virtual environment
+     ```
+     pip install -r requirements.txt --allow-all-external
+     ```
+  * Verify if  xmltodict and requests libraries are installed, if not:
+     ```
+     pip install xmltodict requests
+     ```
+
+#### Requirements to fabric
+    ```
+     yum install gcc python-devel
+    ```
+##### Requeriments for mongoDB
+    ```
+     pip install pymongo
```

Comment:
I think this requeriment is leftover in the readme, in fact this library exists in requeriments.txt:
pymongo==2.7.2